src/store/store.ts

```diff
@@ -20,8 +21,30 @@ export const storeConfig = configureStore({
       },
     }),
 })
+
+export interface Card {
+  columnID: string
+  text: string
+}
+
+export interface Column {
+  title: string
+}
+
+export type WithID<T extends object> = T & { id: string }
```

Comment:
This type is not related to `store` itself, it's rather helper. My suggestion is to move it to utility or helpers folder. You can create, for example, `src/helpers/types.ts` file for that.